scripts/pull_github_pr.sh

```diff
@@ -70,10 +74,15 @@ check_jenkins_job_status() {
     ORANGE='\033[0;33m'
     NC='\033[0m'
     echo -e "${ORANGE}\nWARNING:${NC} $jenkins_job is not stable"
-    exit 1
+    echo "exiting script since gating is unstable, you can force it by running ./script/pull_github_pr.sh <pr num> --force"
```

Comment:
Nitpick: would be nice to get a link to the job which you consider unstable. As a maintainer, I might want to take a look into that, but not remember exactly where that job is.

By the way, I hope that your script actually works on various branches (open-source, enterprise, different branches, etc.) and NOT JUST ONE MASTER. We use the same script on all branches... If it doesn't work on other branches, another followup will be necessary.

Reply:
Fixed

Reply:
it will work on master and OSS releases. Once this gets merged to `enterprise` I will modify it for enterprise releases. But I assume we are not going to backport this, right?

Reply:
I was under the impression (but maybe @avikivity can correct me I'm wrong) that the enterprise script is identical, it just recognizes that the current working directory is an enterprise working directory. If we start having N different versions of this script, it will just make our life harder going forward any time we need to improve this script...

Backports is not the important issue - consider in a year's time this same script will be in branch-7.0, branch-7.1, enterprise-2025.1, and so on - and it will need to work correctly. Instead of hacking it separately on every branch it can - and probably should - recognize in which branch it is. I think this is what this script has been doing so far (but @avikivity can correct me if I'm wrong).

Reply:
By the way, I work with exactly the same script also to merge Seastar patches - because also in Seastar we use the same merge process! It is really sad to see that won't work any more because of some irrelevant Jenkins verification.
@avikivity what script do you use to merge Seastar patches?